user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+git submodule update --depth 1 -q --init libs/unordered <3>
+python tools/boostdep/depinst/depinst.py -X test -g "--depth 1" unordered <4>
+----
+
+<1> Download the Boost super-project, which contains references to all Boost libraries
+<2> Initialize boost:boostdep[] used to determine internal boost dependencies
+<3> Initialize boost:unordered[].
+Repeat this step for any other modules you need.
+<4> Initialize any dependencies of boost:unordered[].
+Repeat this step for any other modules you need.
+
+This procedure requires `git` and `python`.
+The following steps in this document apply to both methods of installing Boost.
+The only difference is what libraries will be available.
+
+=== Bootstrap
+
+If you followed the steps above, your current working directory should be `{release_basename}`.
+Otherwise, open up a command line terminal, and navigate to the boost folder you extracted.
+
+[tabs,sync-group-id=build]
+====
+Boost.Build::
++
+--
+The Boost libraries are built using a custom build app called boost:build[].
+This app itself is built by running the bootstrap script.
+
+This is the usual procedure to configure, build and install Boost with boost:build[]:
+
+Decide where you are going to install the libraries to, most locations are OK, something like *Documents/Boost-libraries* is fine.
+If your location has a space in the name, you will need to surround the path in quotes in the commands that follow.
+
+Windows:
+
+[source]
+----
+bootstrap.bat
+b2
+b2 install --prefix=C:\Boost
+----
+
+If you get an error during the build process such as the following, you probably do not have an up to date and full installation of pass:[C++] on your machine.
+Go back to <<Prerequisites>> and install or update your version of Visual Studio.
+
+image::cpp-runtime-error.png[]
+
+Unix variants:
+
+[source]
+----
+./bootstrap.sh
+./b2
+./b2 install --prefix=/usr/local
+----
+
+You can adjust the installation `prefix` to your preference in this command.
+
+NOTE:: We break the build and install into separate steps, in order to get better diagnostics when building the libraries, and to suppress too many messages when installing the libraries.
+The single command `.\b2.exe --prefix=<YOUR-INSTALL-FOLDER> -d0 install` will both build and install, but suppresses too many messages during the build process.
+If something goes wrong, it will probably be during the build process, and we want to know about it.
+
+The build process creates all variants of each library.
+For example, there may be _Release_ and _Debug_ variants, or possibly _Multithreaded_ and _Static_ runtime variants.
+Also, libraries that are _header-only_ (in other words, do not require being built) are copied to your installation folder.
+
+TIP: Consider using the `--show-libraries` and `--with-libraries=library-name-list` options if you want to limit the wait instead of building everything.
+--
+
+CMake::
++
+--
+You can also build the Boost libraries with CMake.
```

Comment:
If I'm not mistaken, if you build Boost with CMake, you end up getting not just a different layout, but a different set of CMake targets. So if you built Boost with b2 and you're using Boost.Asio, you will link against `Boost::headers`, while if you built it with CMake, you will link against `Boost::asio`. I see 1.82 version of cmake will also provide `Boost::headers` for compatibility.

I'm not sure whether we're really prepared to include cmake at this point - wouldn't just be cleaner to stick to the official method for now (you can then use Windows/Unix tabs), then attempt to make it "official" for 1.83 and change it?

Reply:
> I'm not sure whether we're really prepared to include cmake at this point

Yes. Your comments totally make sense. I'm taking this into account and will talk about different layouts in the docs. But we're working on cppalliance/boost which will contain (it already does) a proper cmake release so I'm looking ahead here. The only limitation ends up being the source layout (because of how libraries are structured), which doesn't really matter that much. The installed layout will work fine.